wagtail/admin/filters.py

```diff
@@ -18,7 +18,7 @@ class DateRangePickerWidget(SuffixedMultiWidget):
     """
 
     template_name = "wagtailadmin/widgets/daterange_input.html"
-    suffixes = ["after", "before"]
+    suffixes = ["from", "to"]
```

Comment:
Hopefully this isn't too disruptive. Using `before` seems to imply that the upper range is exclusive (e.g. `date_before=2024-03-22` won't return items with `date == 2024-03-22`. I tested this with django-filter and it seems to be inclusive, as they actually combine the date with `time.max`:

https://github.com/carltongibson/django-filter/blob/690a906c3dcc533a6097d2064aa1e5c73544ca55/django_filters/fields.py#L52-L59

This also aligns better with the input placeholders ("Date from" and "Date to").